src/Scene.js

```diff
@@ -15,6 +15,61 @@ type SceneProps = {
 };
 
 class Scene extends React.Component<SceneProps, {}> {
+
+    drawGrid(numRow: number, numColumn: number, width: number, minX: number, minY: number) {
+        const grid = [];
+        let xOffset = minX;
```

Comment:
Within the block for the rows, `xOffset` never changes. The math would read more straightforwardly if we used `minX` directly, without the indirection to another variable.

And likewise for `yOffset` in the columns block.